internal/service/service_test.go

```diff
@@ -0,0 +1,137 @@
+package service_test
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+	"time"
+
+	"github.com/andreipimenov/golang-training-2021/internal/mock"
```

Comment:
It would be more readable if you sort and separate project's imports and external libs ones.